 2019-09-29 02:16:58,999 [P2564/D2/T140] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-28 02:16:58
 2019-09-29 06:16:59,013 [P2564/D2/T91] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-28 06:16:59
 2019-09-29 07:07:25,185 [P2564/D2/T167] ERROR Umbraco.Web.Scheduling.KeepAlive - Failed (at "http://83.41.87.157:8008/umbraco").
System.Net.Http.HttpRequestException: Error al enviar la solicitud. ---> System.Net.WebException: No es posible conectar con el servidor remoto ---> System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder 83.41.87.157:8008
   en System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   en System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   en System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   en System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()
 2019-09-29 07:12:46,530 [P2564/D2/T167] ERROR Umbraco.Web.Scheduling.KeepAlive - Failed (at "http://83.41.87.157:8008/umbraco").
System.Net.Http.HttpRequestException: Error al enviar la solicitud. ---> System.Net.WebException: No es posible conectar con el servidor remoto ---> System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder 83.41.87.157:8008
   en System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   en System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   en System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   en System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()
 2019-09-29 07:18:07,623 [P2564/D2/T111] ERROR Umbraco.Web.Scheduling.KeepAlive - Failed (at "http://83.41.87.157:8008/umbraco").
System.Net.Http.HttpRequestException: Error al enviar la solicitud. ---> System.Net.WebException: No es posible conectar con el servidor remoto ---> System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder 83.41.87.157:8008
   en System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   en System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   en System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   en System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()
 2019-09-29 07:22:43,217 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Terminating
 2019-09-29 07:22:43,237 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Waiting for tasks to complete
 2019-09-29 07:22:43,249 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Terminating
 2019-09-29 07:22:43,249 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Waiting for tasks to complete
 2019-09-29 07:22:43,249 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Terminating
 2019-09-29 07:22:43,249 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Waiting for tasks to complete
 2019-09-29 07:22:43,249 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Tasks completed, terminated
 2019-09-29 07:22:43,249 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Terminating
 2019-09-29 07:22:43,265 [P2564/D2/T78] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Tasks completed, terminated
 2019-09-29 07:22:43,265 [P2564/D2/T48] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Tasks completed, terminated
 2019-09-29 07:22:43,265 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Waiting for tasks to complete
 2019-09-29 07:22:43,265 [P2564/D2/T48] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Tasks completed, terminated
 2019-09-29 07:22:43,265 [P2564/D2/T161] INFO  Umbraco.Core.MainDom - Stopping...
 2019-09-29 07:22:43,280 [P2564/D2/T161] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2019-09-29 07:22:43,295 [P2564/D2/T161] INFO  Umbraco.Web.ExamineStartup - Complete (took 23ms)
 2019-09-29 07:22:43,311 [P2564/D2/T161] INFO  Umbraco.Core.MainDom - Released MainDom.
 2019-09-29 07:22:43,311 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Terminating
 2019-09-29 07:22:43,311 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Waiting for tasks to complete
 2019-09-29 07:22:43,311 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Terminating
 2019-09-29 07:22:43,311 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Waiting for tasks to complete
 2019-09-29 07:22:43,311 [P2564/D2/T173] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Tasks completed, terminated
 2019-09-29 07:22:43,311 [P2564/D2/T173] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Tasks completed, terminated
 2019-09-29 07:22:43,389 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Terminating
 2019-09-29 07:22:43,389 [P2564/D2/T161] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Waiting for tasks to complete
 2019-09-29 07:22:43,389 [P2564/D2/T48] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Tasks completed, terminated
 2019-09-29 07:22:43,498 [P2564/D2/T161] INFO  Umbraco.Core.UmbracoApplicationBase - Application shutdown. Details: HostingEnvironment

_shutDownMessage=HostingEnvironment initiated shutdown
HostingEnvironment originó el cierre

_shutDownStack=   en System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   en System.Environment.get_StackTrace()
   en System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()
   en System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()
   en System.Web.Hosting.PipelineRuntime.StopProcessing()
 2019-09-29 15:18:53,282 [P764/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.15.3 application starting on SYNOVMW10-UMBRA
 2019-09-29 15:18:53,358 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-09-29 15:18:53,438 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 75ms)
 2019-09-29 15:18:53,595 [P764/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-09-29 15:18:53,595 [P764/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-09-29 15:18:53,607 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-29 15:18:53,842 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 246ms)
 2019-09-29 15:18:53,842 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-09-29 15:18:53,860 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 8ms)
 2019-09-29 15:18:54,014 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-29 15:18:54,014 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-09-29 15:18:54,014 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-09-29 15:18:54,014 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 1ms)
 2019-09-29 15:18:54,014 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-29 15:18:54,014 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-09-29 15:18:54,014 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-09-29 15:18:54,029 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 7ms)
 2019-09-29 15:18:54,047 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-29 15:18:54,047 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-09-29 15:18:54,047 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-09-29 15:18:54,047 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 2ms)
 2019-09-29 15:18:54,047 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-29 15:18:54,047 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-09-29 15:18:54,047 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-09-29 15:18:54,065 [P764/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 3ms)
 2019-09-29 15:19:03,344 [P764/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-09-29 15:19:04,045 [P764/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-09-29 15:19:04,474 [P764/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-09-29 15:19:04,474 [P764/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-09-29 15:19:04,703 [P764/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 11719ms)
 2019-09-29 15:19:05,732 [P764/D2/T11] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://83.41.85.111:8008/umbraco
 2019-09-29 15:19:05,732 [P764/D2/T11] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://83.41.85.111:8008/umbraco (UmbracoModule request)
 2019-09-29 15:19:07,346 [P764/D2/T11] INFO  umbraco.content - Load Xml from file...
 2019-09-29 15:19:07,373 [P764/D2/T11] INFO  umbraco.content - Loaded Xml from file.
 2019-09-29 15:19:10,889 [P764/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-29 15:19:10,889 [P764/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-09-29 15:19:10,889 [P764/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-09-29 15:19:10,905 [P764/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 4ms)
 2019-09-29 15:19:12,890 [P764/D2/T11] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-29 15:20:07,514 [P764/D2/T5] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-29 15:20:07,514 [P764/D2/T5] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-09-29 15:20:07,514 [P764/D2/T5] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-09-29 15:20:07,514 [P764/D2/T5] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 2ms)
 2019-09-29 15:20:07,545 [P764/D2/T12] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-28 15:20:07
 2019-09-29 19:20:07,592 [P764/D2/T70] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-28 19:20:07
 2019-09-29 23:20:07,607 [P764/D2/T37] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-28 23:20:07
